blog/posts/20251202-savetheclocktower-pulsar-on-electron-30.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Pulsar on Electron 30: what it means for you"
+author: savetheclocktower
+date: 2025-12-02
+category:
+  - dev
+---
```

Comment:
Partly for nostalgia, but I suppose for findability of posts as well, consider one or both of these tags that appear on one or more of the other Electron modernization blog posts? (Posts such as https://blog.pulsar-edit.dev/posts/20240124-mauricioszabo-the-quest-for-electron-lts/ and https://blog.pulsar-edit.dev/posts/20250630-savetheclocktower-pulsar-next-testing/)

```suggestion
category:
  - dev
tag:
  - modernization
  - electron
---
```